apsuite/commissioning_scripts/measure_id_integrals.py

```diff
@@ -16,33 +17,39 @@
 class IDParams:
     """."""
 
-    def __init__(self, phases, meas_type):
+    def __init__(self, phases, meas_type, sofb):
         """."""
         self.phases = phases
         self.meas_type = meas_type
         if self.meas_type == MeasIDIntegral.MEAS_TYPE.Static:
             self.phase_speed = 0.5
-            self.sofb_mode = 'SlowOrb'
+            self.sofb_mode = sofb.data.SOFBMode._fields[1]
```

Comment:
If you want the string here, it is better to do:
`sofb.data.SOFBMode._fields[sofb.data.SOFBMode.SlowOrb]`

Reply:
done, thanks